### ISSUE TITLE
New /api/v1/logs endpoint in GW 3.1-rc6

### ISSUE THREAD

**Issue by srt** [zweckj/pylamarzocco#7]
The new version 3.1-rc6 of the gw fw seems to support a new endpoint for logs that is missing in the former v2.2-rc0.

```
strings home-gateway_v3.1-rc6.bin | grep /v1/           
/api/v1/config
/api/v1/command/*
/api/v1/streaming
/api/v1/logs
```

Sending a GET request to that url causes the connection to be closed (instead of a 404 for an invalid uri).
I am not sure if that endpoint is of any use but maybe we can find out :-)

`/api/v1/command/*` also looks kind of promising.

**Comment by zweckj** [comment]
> `/api/v1/command/*` also looks kind of promising.

those I'd be **very** interested in. I don’t think they are being used though, I didn’t see any traffic in that direction…

**Comment by rccoleman** [comment]
Yeah, unfortunately short of random fuzzing the APIs, I/we have mainly relied on monitoring what the mobile app does.  I do hope that they have plans to expand local control and we're just seeing the start, but that would kinda buck the industry trend toward cloud services.

**Comment by srt** [comment]
The (cordova) app only seems to only use streaming and config (in `/assets/www/build/main.js`). The js file is minified but kinda readable.

Maybe its time to have a closer look at the actual gateway firmware...

Regarding MQTT the machine connects to `mqtt.lamarzocco.io` and (un)fortunately checks the certificate. Which is btw. the reason why they are currently pushing out the new firmware as older machines seems to have issues with this after Sept. 10 2023. (see https://www.kaffee-netz.de/threads/la-marzocco-linea-micra.147919/page-96#post-2255661)

**Comment by zweckj** [comment]
closing this, as this is no longer applicable with the new FW
